packages/shared/styles/components/molecules/SfBanner.scss

```diff
@@ -2,80 +2,82 @@
 
 .sf-banner {
   box-sizing: border-box;
-  display: flex;
-  min-height: var(--banner-min-height, 9rem);
-  padding: var(--banner-padding, var(--spacer-big));
-  color: var(--banner-color, var(--c-text));
-  background-repeat: no-repeat;
-  background-position: var(--banner-background-position, bottom left);
-  background-size: cover;
-  background-color: var(--banner-background);
+  display: var(--banner-display, flex);
+  flex-direction: var(--banner-flex-direction, column);
+  justify-content: var(--banner-justify-content, left);
+  background-color: var(--background-color);
+  background-position: var(--banner-background-position, center);
+  background-repeat: var(--banner-background-repeat, no-repeat);
+  background-size: var(--banner-background-size, cover);
   background-image: var(--background-image);
+  color: var(--banner-color, var(--c-text));
+  width: var(--banner-size, var(--banner-width, 100%));
+  min-height: var(--banner-size, var(--banner-min-height, 14rem));
+  letter-spacing: var(--banner-letter-spacing, 0.1px);
   &__container {
-    flex: var(--banner-container-flex, 0 0 100%);
+    display: flex;
+    justify-content: left;
+    flex-direction: column;
+    margin: var(--banner-container-margin, 3.125rem 0 0 3.125rem);
   }
   &__subtitle {
-    margin: var(--banner-subtitle-margin);
-    color: var(--banner-subtitle-color, var(--c-dark-variant));
-    font: var(
+    @include font(
       --banner-subtitle-font,
-      font(
-        --banner-subtitle,
-        300,
-        var(--font-size-regular),
-        1.16,
-        var(--font-family-primary)
-      )
+      var(--font-normal),
+      var(--font-2xs),
+      1.4,
+      var(--font-family-secondary)
+    );
+    color: var(
+      --banner-color,
+      var(--banner-subtitle-color, var(--c-text-muted))
```

Comment:
try this color ` --c-dark-variant` I think it will fit to design, wdyt ? And also, subtitle should be written in capital letters, use variable to give possibility to overwrite it :)